hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/audit/SCMAction.java

```diff
@@ -32,6 +32,7 @@
   GET_CONTAINER_WITH_PIPELINE,
   LIST_CONTAINER,
   LIST_PIPELINE,
+  GET_PIPELINE,
```

Comment:
Usually we add new fields at the end of enums.

Reply:
Sure, I can move it to the end.

Reply:
I think it is okay for now as we don't have to maintain compatibility.